css/css-flexbox/gap-007-ltr.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Flexbox Test: gap - wrap vertical</title>
```

Comment:
"vertical" here means a vertical writing-mode? or that the gap is vertical?

Reply:
gap is vertical 👍